skyportal/handlers/api/source.py

```diff
@@ -232,17 +240,25 @@ def get(self, obj_id=None):
             )
 
             return self.success(data=source_info)
-        q = Obj.query.filter(
-            Obj.id.in_(
-                DBSession()
-                .query(Source.obj_id)
-                .filter(
-                    Source.group_id.in_(
-                        [g.id for g in self.current_user.accessible_groups]
-                    )
+
+        # Instead of single source, let's try to get all of them
```

Comment:
What does this comment mean? Perhaps you misunderstood the original query - it was always querying all matching Objs

Reply:
The way I understood it, the first IF statement tests for a specific obj_id and returns a single match. Then if that doesn't happen we go to the "get all sources" branch. 

I think it is good to put a comment (maybe a differently phrased one) to show this is a completely different option that is not reached if the first IF is triggered (due to a "return" in that branch).

Reply:
Should be obvious from the indentation and return statement, or can add a comment like this 
```suggestion
        # Fetch multiple sources
```

Reply:
You and I are in an excellent place to say what is obvious to someone who already knows this code very well and to someone who is looking at it for the first time... 

That being said, I've changed the comment to what you suggested!